File: SDKs/Aspose.Pdf_Cloud_SDK_For_Java/src/main/java/com/aspose/pdf/model/AppendDocument.java
package com.aspose.pdf.model;

public class AppendDocument {
  private String Document = null;
  private Integer StartPage = null;
  private Integer EndPage = null;
  /**
	 * getDocument
	 * Gets String
	 * @return Document
	 */
  public String getDocument() {
    return Document;
  }

	/**
	 * setDocument
	 * Sets String
	 * @param Document String
	 */
  public void setDocument(String Document) {
    this.Document = Document;
  }

  /**
	 * getStartPage
	 * Gets Integer
	 * @return StartPage
	 */
  public Integer getStartPage() {
    return StartPage;
  }

	/**
	 * setStartPage
	 * Sets Integer
	 * @param StartPage Integer
	 */
  public void setStartPage(Integer StartPage) {
    this.StartPage = StartPage;
  }

  /**
	 * getEndPage
	 * Gets Integer
	 * @return EndPage
	 */
  public Integer getEndPage() {
    return EndPage;
  }

	/**
	 * setEndPage
	 * Sets Integer
	 * @param EndPage Integer
	 */
  public void setEndPage(Integer EndPage) {
    this.EndPage = EndPage;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class AppendDocument {\n");
    sb.append("  Document: ").append(Document).append("\n");
    sb.append("  StartPage: ").append(StartPage).append("\n");
    sb.append("  EndPage: ").append(EndPage).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Pdf-Cloud-SDK-for-.NET/src/Com/Aspose/PDF/Model/AppendDocument.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class AppendDocument {
    public string Document { get; set; }

    public int? StartPage { get; set; }

    public int? EndPage { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class AppendDocument {\n");
      sb.Append("  Document: ").Append(Document).Append("\n");
      sb.Append("  StartPage: ").Append(StartPage).Append("\n");
      sb.Append("  EndPage: ").Append(EndPage).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
